### PR TITLE
CIS-88 Updated `Reachability.init` in InternetConnection

### PR DESCRIPTION
**Pull request by buh**
Removed `Reachability.init` with our site URL (impossible case).

### CODE REVIEW

File: Sources/Client/Utils/InternetConnection.swift [hunk 1]
```diff
@@ -50,7 +50,14 @@ public final class InternetConnection {
     private var lastState: State = .unknown
     private lazy var reachability: Reachability? = {
         do {
-            let reachability = try Reachability(hostname: Client.shared.baseURL.wsURL.host ?? "getstream.io")
```
Comment (b-onc) [inline Sources/Client/Utils/InternetConnection.swift:53]: It should be
```swift
let reachability = try Reachability(hostname: Client.shared.baseURL.wsURL.host ?? "getstream.com")
```
per task description, not empty init

Comment (buh) [reply Sources/Client/Utils/InternetConnection.swift:53]: We don't have getstream.com

Comment (b-onc) [reply Sources/Client/Utils/InternetConnection.swift:53]: @tbarbugli I thought we should use `getstream.com`?

**Comment by buh** [comment]
@tbarbugli @b-onc can you check this PR?

**Pull request merged**